competency-explorer/src/helpers/loadFile.ts

```diff
@@ -0,0 +1,16 @@
+import { MANAGER_COMPETENCIES, IC_COMPETENCIES } from './constants';
+import { loadFile as lf } from '@toolkip/server';
+
+const loadFile = async (filename: string) => {
+  return await lf({ filename });
```

Comment:
i had some difficulty with `fetch` not actually loading the file appropriately, so I went for my library's `GET` version

Reply:
Out of curiosity, what difficulty was that?

Reply:
it failed to resolve the relative path when hosted through GH pages

Reply:
what a jerk